specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/definitions.json

```diff
@@ -264,10 +264,10 @@
       "properties": {
         "keyName": {
           "type": "string",
-          "description": "Gets or sets the agent registration key name - Primary or Secondary.",
+          "description": "Gets or sets the agent registration key name - primary or secondary.",
           "enum": [
-            "Primary",
-            "Secondary"
+            "primary",
```

Comment:
!

Reply:
The Spec was incorrect here and I just changed it to reflect what we are sending from the service. And we are in the process of releasing new Automation SDKs based on Swagger and are still in preview mode.

Our last stable release of SDK was based on Hyak.